doc/design/katib_on_sqlflow.md

```diff
@@ -64,28 +65,59 @@ We need to develop a new codegen, `codegen_couler.go`, for SQLFlow.  `codegen_co
 
 ### The Integration via Couler
 
-SQLFlow parses each SQL program into an IR, which is a list of statement IRs.  The `codegen_couler.go` converts the IR into a Couler program.   We need to add a Couler functions `couler.katib.train` for the calling by the generated Couler program.
+SQLFlow parses each SQL program into an IR, which is a list of statement IRs.  The `codegen_couler.go` converts the IR into a Couler program.   We need to add a Couler functions `couler.sqlflow.train` for the calling by the generated Couler program.
 
 Consider the following example program.
 
 ```sql
 SELECT * FROM a, b WHERE a.id = b.id INTO c;
-SELECT * FROM c TO TRAIN model_def WITH objective=multi:softmax, eta=1 LABEL class INTO my_xgb_model;
+SELECT * FROM c TO TRAIN data_scientist/xgboost:v0.5/xgboost.gbtree 
```

Comment:
Maybe we can add attribute `range.max_depth=[2, 10]` in the demo so that users can know it's an auto hyperparameter search job.

Reply:
It may be a way to label it is a katib training job. However, the concept of auto model training is users need not specify range for any parameter (if they want, they can), we need to fill default value instead.

I still do not find a good way to label it is a katib training job in sql statement. Any suggestions?

Reply:
Maybe we can use env to label it, like `SQLFLOW_XGBOOST_SUBMITTER=katib`